lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu 
+  def initialize
+    @order = []
+    @menu_hash = {
+      "kung po chicken" => 7,
+      "crispy chilli beef" => 9,
+      "crispy duck pancakes" => 8,
+      "crispy seaweed" => 2,
+      "spare ribs" => 5,
+      "egg fired rice" => 4,
+      "chow mein" => 6
+    }
+  end
+
+  def view_menu
+    @menu_hash.each_with_index do |(key, value), index|
+      puts "number: #{index} | #{key} | £#{value}"
```

Comment:
Good use of the hash method to print the menu!
Just one things: I can see the first 'number: index' that is printed is 'number: 0'. Is it really what you wanted? Wouldn't it be better to start from 1?